docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,185 @@
+
+If you deploy your application (client) on another environment than the Kubernetes cluster for Scalar products (e.g., you deploy your application on another  Kubernetes cluster, container platform, or server), the application can access Scalar products using a Load Balancer provided by each cloud service.
+
+You can create the Load Balancer to set `envoy.service.type` to `LoadBalancer` in your custom values file. When you create it, you can access Scalar Envoy (A service resource of Kubernetes) via created Load Balancer. You can also configure the Load Balancer configurations using annotations. Please refer to the following document for more details on how to configure your custom values file.
+
+* [Service configurations](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-envoy.md#service-configurations)
+
+If you use the Load Balancer, you need to set the FQDN or IP address of the Load Balancer in the properties file for the application (client) as follows.
+
+* Client properties file for ScalarDB Server
+  ```properties
+  scalar.db.contact_points=<FQDN or IP address of Load Balancer>
+  scalar.db.contact_port=60051
+  scalar.db.storage=grpc
+  scalar.db.transaction_manager=grpc
+  ```
+* Client properties file for ScalarDL Ledger
+  ```properties
+  scalar.dl.client.server.host=<FQDN or IP address of Load Balancer>
+  scalar.dl.ledger.server.port=50051
+  scalar.dl.ledger.server.privileged_port=50052
+  ```
+* Client properties file for ScalarDL Ledger and ScalarDL Auditor (Auditor mode)
+  ```properties
+  # Ledger
+  scalar.dl.client.server.host=<FQDN or IP address of Load Balancer>
+  scalar.dl.ledger.server.port=50051
+  scalar.dl.ledger.server.privileged_port=50052
+  
+  # Auditor
+  scalar.dl.client.auditor.enabled=true
+  scalar.dl.client.auditor.host=<FQDN or IP address of Load Balancer>
+  scalar.dl.auditor.server.port=40051
+  scalar.dl.auditor.server.privileged_port=40052
+  ```
+
+The concrete implementation of the Load Balancer and access method depend on the Kubernetes cluster. If you use a managed Kubernetes cluster, please refer to the following cloud provider's official documents for more details.
+
+### EKS
+
+* [Network load balancing on Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
+
+### AKS
+
+* [Use a public standard load balancer in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/load-balancer-standard)
+* [Use an internal load balancer with Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/internal-lb)
+
+## Access Scalar products from a bastion server (For testing purposes only / Not recommended in the production environment)
+
+You can access Scalar products from a bastion server using the `kubectl port-forward` command. If you create a ScalarDL Auditor mode environment, you need to access two Kubernetes clusters from one bastion server to access ScalarDL this way.
```

Comment:
```suggestion
You can run client requests to ScalarDB or ScalarDL from a bastion server by running the `kubectl port-forward` command. If you create a ScalarDL Auditor mode environment, however, you must run two Kubernetes clusters from one bastion server to access ScalarDL.
```

Reply:
Understood. Your explanation makes perfect sense to me! 

How about we use the following?

```suggestion
You can run client requests to ScalarDB or ScalarDL from a bastion server by running the `kubectl port-forward` command. If you create a ScalarDL Auditor mode environment, however, you must run two `kubectl port-forward` commands with different kubeconfig files from one bastion server to access two Kubernetes clusters.
```

Reply:
Looks good to me! 
Thank you for updating your suggestion.
I apply this suggestion.